packages/libs/eda/src/lib/core/components/visualizations/implementations/LineplotVisualization.tsx

```diff
@@ -2797,7 +2799,7 @@ export function AggregationInputs<F extends string, P extends Array<string>>(
   props: AggregationConfig<F, P>
 ) {
   const classes = useInputStyles();
-
+  console.log(props);
```

Comment:
Can you undo this change, please? Thanks!

Reply:
Oops - yes of course. Thanks for spotting!